prediction_market_agent/agents/microchain_agent/app.py

```diff
@@ -255,15 +269,39 @@ def get_function_bullet_point_list(agent: Agent, model: str) -> str:
     else:
         st.markdown("The agent is not initialized yet.")
 
+
+def on_click_run_agent() -> None:
```

Comment:
Could you move this to the top section with the rest of the functions, just for readability of the streamlit app code?